feature/gribi/otg_tests/ipv4_entry_test/ipv4_entry_test.go

```diff
@@ -155,7 +160,7 @@ func TestIPv4Entry(t *testing.T) {
 				fluent.IPv4Entry().WithNetworkInstance(*deviations.DefaultNetworkInstance).
 					WithPrefix(dstPfx).WithNextHopGroup(nhgID),
 			},
-			wantGoodFlows: []*ondatra.Flow{ecmpFlow},
+			wantGoodFlows: []string{"ecmpFlow"},
```

Comment:
why move the flows to being defined below?

Reply:
I think you're referring to why we're defining flows in the validate func and not here. There are 2 reasons: 
- in OTG we don't have a way to clear the statistics before running the traffic. That is being done automatically only when we start the traffic on all flows.
- ate.OTG().StartTraffic(t) doesn't have an option to start traffic on individual flows.
So, considering the above, we can't have for example 3 flows defined and we start just 2, because the stats will not be cleared between runs.